tests/common/member_test.go

```diff
@@ -74,34 +75,27 @@ func TestMemberAdd(t *testing.T) {
 	}
 
 	quorumTcs := []struct {
-		name                string
-		strictReconfigCheck bool
-		waitForQuorum       bool
-		expectError         bool
+		name                       string
+		disableStrictReconfigCheck bool
```

Comment:
It's anti-pattern, `not disable == enable`. suggest to rename to `enableStrictReconfigCheck` or just `strictReconfigCheck`

Reply:
I've just remembered this and renamed it to `strictReconfigCheck`. I've force pushed.